float32 mass
geometric_msgs/Vector3 dimension